### PR TITLE
Change document title per page using React component

### PR DESCRIPTION
**Pull request by gilleswittenberg**

### CODE REVIEW

File: app/components/DocumentTitle/__tests__/DocumentTitle.test.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { render, cleanup } from 'react-testing-library';
+
+import DocumentTitle from '..';
+
+describe('DocumentTitle', () => {
+  afterEach(cleanup);
```
Comment (petercuret) [inline app/components/DocumentTitle/__tests__/DocumentTitle.test.js]: afterEach(cleanup) can be removed if there is only test.

File: app/components/DocumentTitle/index.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { useEffect } from 'react';
+
+const DocumentTitle = ({ title }) => {
+  useEffect(() => {
+    document.title = title;
+  }, [title]);
```
Comment (petercuret) [inline app/components/DocumentTitle/index.js]: You could add a return function to this effect, which sets the document title back to a default when the component is removed. 

This is useful when another page is rendered which doesn't contain a DocumentTitle. In that case it won't show the wrong page title, but just a default.

**Review by petercuret — CHANGES_REQUESTED** [review]
Looks good! I left two small comments to have a look at.

File: app/components/DocumentTitle/index.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { useEffect } from 'react';
+
+const defaultTitle = 'Registratie';
+
+const DocumentTitle = ({ title }) => {
```
Comment (janjaap) [inline app/components/DocumentTitle/index.js]: Why not use [react helmet](https://github.com/nfl/react-helmet)?

File: app/components/DocumentTitle/__tests__/DocumentTitle.test.js (hunk 1 above)
Comment (gilleswittenberg) [reply app/components/DocumentTitle/__tests__/DocumentTitle.test.js]: Done (https://github.com/Amsterdam/registraties/pull/107/commits/66193688f624ffd138fef74db0d54969d46e0aca)

File: app/components/DocumentTitle/index.js (hunk 3 above)
Comment (gilleswittenberg) [reply app/components/DocumentTitle/index.js]: Overkill in my eyes. But can look into it a bit more. Also cause I've just realised that internationalisation should also be supported while changing titles.

File: app/components/DocumentTitle/index.js (hunk 2 above)
Comment (gilleswittenberg) [reply app/components/DocumentTitle/index.js]: Done (https://github.com/Amsterdam/registraties/pull/107/commits/45fef4073bbc992f4437dd6d106db9914466f83f)

**Review by janjaap — APPROVED** [review]

**Review by petercuret — APPROVED** [review]

**Pull request merged**